lib/mailjet/resource.rb

```diff
@@ -201,6 +205,34 @@ def map_keys(hash, method)
           _hash
         end
       end
+      
+      def change_resource_path(options = {})
+        ver = choose_version(options)
+        url = Mailjet.config.end_point
+        call = Mailjet.config.perform_api_call
+        if options != {}
+          if options['call'] == false || options['call'] == true
```

Comment:
I can't use "present" as it doesn't make any distinction between false and nil, and I don't have any else.